### ISSUE TITLE
Formatting one file dirties another file unecessarily

### ISSUE THREAD

**Issue by DevMude** [ajmnz/prisma-import#42]
I have two prisma files

1\. `Company.prisma`
```prisma
import { CompanyDetails } from "../CompanyDetails/CompanyDetails"

model Company {
  id      String          @id
  name    String
  slug    String          @unique
  details CompanyDetails?
}
```
2\. `CompanyDetails.prisma`
```prisma
import { Company } from "../Company/Company"

model CompanyDetails {
  companyId  String @id
  shortBio   String
  company Company @relation(fields: [companyId], references: [id])
}
```

When I manually format `CompanySchema.prisma`, it leaves 1 blank newline at the end of the file, but then it automatically formats `CompanyDetailsSchema.prisma` with 2 blank newlines at the end of file. Same thing happens when I go to manually format `CompanyDetailsSchema.prisma`. So I format one file, it dirties the other. Then I format the other file and it dirties the original. Ad infinitum.

My vscode `settings.json`:
```json
{
  "[prisma]": {
    "editor.formatOnSave": true,
    "editor.defaultFormatter": "ajmnz.prisma-import"
  }
}
```

**Comment by ajmnz** [comment]
Could you please share which OS and VS Code version you're on?

**Comment by DevMude** [comment]
@ajmnz I've just noticed this problem doesn't exist on my Mac, only on my Windows PC. Will update this comment with specific versions when I can access my Windows PC again.

**Comment by DevMude** [comment]
@ajmnz 

OS: Windows 10.0.19044 Build 19044
VS Code: v1.77.3

**Comment by ajmnz** [comment]
@DevMude 👋

This issue has been fixed via #47 and is now available in version `4.13.0`.
Please give it a try and let me know if it has been resolved.

**Comment by DevMude** [comment]
@ajmnz Works like a charm! Thanks mate, super helpful fix :)
